### PR TITLE
Fixing Get Started header link

### PR DESCRIPTION
**Pull request by rszper**
Get started on the top nav should go to https://beam.apache.org/get-started

See https://github.com/apache/beam/issues/21537

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by rszper** [comment]
R: @sachinag

**Comment by rszper** [comment]
R: @alevtinaboiko
R: @bullet03

**Comment by damccorm** [comment]
Run Website_Stage_GCS PreCommit

File: website/www/site/layouts/partials/header.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
           <a class="navbar-link" href={{ "/about" | relLangURL }}>{{ T "nav-about" }}</a>
         </li>
         <li>
-         <a class="navbar-link" href={{ "/get-started/beam-overview/" | relLangURL }}>{{ T "nav-get-started" }}</a>
+         <a class="navbar-link" href={{ "/get-started/" | relLangURL }}>{{ T "nav-get-started" }}</a>
```
Comment (damccorm) [inline website/www/site/layouts/partials/header.html:60]: It looks like you just changed this for the mobile nav (or something else?). This didn't seem to fix the main desktop version - http://apache-beam-website-pull-requests.storage.googleapis.com/23490/index.html - it looks like there's another reference that you can update on line 19 - https://github.com/apache/beam/blob/3248486a25f905c438b8dd41282bddc71410b467/website/www/site/layouts/partials/header.html#L19

Comment (rszper) [reply website/www/site/layouts/partials/header.html:60]: Turns out it was line 128 that made the difference on the desktop version.

**Review by damccorm — APPROVED** [review]
LGTM, thanks! I noticed some other references to the beam-overview page (https://github.com/apache/beam/search?p=1&q=beam-overview), but on inspection they all looked valid to me.

**Pull request merged**